recipe/meta.yaml

```diff
@@ -30,24 +32,25 @@ test:
     - tests
   requires:
     - pytest >=6.2,<7
-    - pytest-asyncio >=0.15,<1
     - pytest-benchmark >=3.4,<4
-    - pytest-cov >=2.11,<3
-    - pytest-describe >=1,<2
-    - pytest-timeout >=1.4,<2
+
     - pip
   commands:
     - pip check
-    - python -c '__import__("os").unlink("tests/test_docs.py")'
-    - pytest --cov graphql --cov-report term-missing:skip-covered --no-cov-on-fail
+    - pytest
 
 about:
   home: https://github.com/graphql-python/graphql-core
+  description: |
+    GraphQL-core 3 is a Python 3.6+ port of GraphQL.js, 
+    the JavaScript reference implementation for GraphQL, 
+    a query language for APIs created by Facebook.
```

Comment:
For readability, `description` can be placed below `summary`

Reply:
Updated this